_includes/integrations/angular-quick-start.md

```diff
@@ -63,22 +63,20 @@ This procedure requires:
     ```html
     <h1>{{site.productname}} {{site.productmajorversion}} Angular Demo</h1>
     <editor
-      initialValue="<p>This is the initial content of the editor</p>"
```

Comment:
Nothing wrong with this but I can't get the "rouge highlighter" to handle it. (The highlights for the remaining code sample would be wrong)